apps/vanilla/src/pages/scripts/index.ts

```diff
@@ -0,0 +1,63 @@
+import { LimitOffsetPagination } from '@js-camp/core/models/limitOffsetPagination';
+
+import { AnimeListItem } from '@js-camp/core/models/anime/animeListItem';
+import { AnimeType } from '@js-camp/core/models/anime/animeType';
+import { AnimeStatus } from '@js-camp/core/models/anime/animeStatus';
+
+import {
+  AnimeService,
+} from '../../../api/services/animeService';
+
+import {
+  getPaginateOptions,
+  updatePagination,
+} from './pagination';
+import { getCurrentSortOptions } from './sorting';
+
+document.addEventListener('DOMContentLoaded', () => {
+  fetchTable(1);
+});
+
+/** Fetch anime list and add it to page.
+ *  @param page Page number.
+ * */
+export async function fetchTable(page: number): Promise<void> {
+  const paginateOptions = getPaginateOptions(page);
+  const animePaginatedList = await AnimeService.getList(
+    paginateOptions,
+    getCurrentSortOptions(),
+  );
+  updateTable(animePaginatedList);
+  updatePagination(animePaginatedList.count, paginateOptions);
+}
+
+/** Update table body.
+ *  @param animePaginatedList Paginated list of anime.
+ * */
+function updateTable(animePaginatedList: LimitOffsetPagination<AnimeListItem>): void {
+  const tableBody = document.querySelector('.anime-table__body') as Element;
```

Comment:
FYI, `document.querySelector` have a generic.
```suggestion
  const tableBody = document.querySelector<Element>('.anime-table__body');
```